### PR TITLE
FB-698 Add dbUpdatedAt to Flow for Orders

### PR DESCRIPTION
**Pull request by zhalninde**

### CODE REVIEW

**Review by zolt — APPROVED** [review]

File: backend-core/src/main/kotlin/com/rarible/flow/core/domain/order.kt [hunk 1]
```diff
@@ -66,6 +66,9 @@ data class Order(
     val platform: FlowOrderPlatformDto? = null
 ) {
 
+    @Indexed
+    val dbUpdatedAt: LocalDateTime = LocalDateTime.now()
```
Comment (atapin) [inline backend-core/src/main/kotlin/com/rarible/flow/core/domain/order.kt:70]: let's use Instant

File: backend-core/src/main/kotlin/com/rarible/flow/core/repository/OrderRepository.kt [hunk 2]
```diff
@@ -55,11 +56,11 @@ class OrderRepositoryCustomImpl(val mongo: ReactiveMongoTemplate): OrderReposito
         return mongo.find(query)
     }
 
-    override suspend fun update(filter: OrderFilter, updateDefinition: UpdateDefinition): UpdateResult {
+    override suspend fun update(filter: OrderFilter, update: Update): UpdateResult {
         return mongo
             .update<Order>()
             .matching(filter.criteria())
-            .apply(updateDefinition)
+            .apply(update.set(Order::dbUpdatedAt.name, LocalDateTime.now()))
```
Comment (atapin) [inline backend-core/src/main/kotlin/com/rarible/flow/core/repository/OrderRepository.kt:63]: this change should go to a migration

**Review by atapin — CHANGES_REQUESTED** [review]

**Review by zolt — APPROVED** [review]

File: backend-core/src/main/kotlin/com/rarible/flow/core/domain/order.kt (hunk 1 above)
Comment (zhalninde) [reply backend-core/src/main/kotlin/com/rarible/flow/core/domain/order.kt]: ок

File: backend-core/src/main/kotlin/com/rarible/flow/core/repository/OrderRepository.kt (hunk 2 above)
Comment (zhalninde) [reply backend-core/src/main/kotlin/com/rarible/flow/core/repository/OrderRepository.kt]: done

**Review by atapin — APPROVED** [review]

**Pull request merged**